lib/l10nUtil.js

```diff
@@ -82,32 +55,57 @@ const braveResourcesComponentsStringsPath = path.resolve(path.join(srcDir, 'brav
 const braveExtensionMessagesPath = path.resolve(path.join(srcDir, 'brave', 'components', 'brave_extension', 'extension', 'brave_extension', '_locales', 'en_US', 'messages.json'))
 const braveRewardsExtensionMessagesPath = path.resolve(path.join(srcDir, 'brave', 'components', 'brave_rewards', 'resources', 'extension', 'brave_rewards', '_locales', 'en_US', 'messages.json'))
 
-// When adding new grd or grd files, never add a grdp part path without a parent grd path.
-// Group them with a leading and trailing newline to keep this file organized.
+// Helper function to find all grdp parts in a grd.
+function getGrdPartsFromGrd(path) {
+  const grd = new JSDOM(fs.readFileSync(path, 'utf8'))
```

Comment:
Just interested whether you considered using a more lightweight xml parser rather than a full DOM parser? Or is it because we happen to include this dep already?

Reply:
I tried using a SAX parser instead at first, but wasn't able to find a synchronous way to use it. From what I understand there's no way to use async/await with a top level assignment. If there is, I will gladly go back to SAX.